### DISCUSSION TITLE
An another small appearance issue.

### DISCUSSION THREAD

**Discussion by Neodevils** [community/community#28160] [Discussions] [answered]
![IMG_20220803_124615](https://user-images.githubusercontent.com/71941576/182578695-2f6a770f-297f-43d5-b66d-27f32232af03.jpg)

**Comment by Neodevils** [comment]
border-left property doesn't seem like centered on avatar.

**Comment by stevepopovich** [comment, marked as answer]
this looks like a problem with the mobile website and not the native mobile app. Moving category to discussions!
